Yoast/wordpress-seo
# Simplify the way of registering upgrade routines

## Summary

This PR can be summarized in the following changelog entry:

* N/A - Refactored the way we do upgrade routines. Makes it easier to extend.

## Relevant technical choices:

*

## Test instructions

This PR can be tested by following these steps:

* Set the version to a lower version (with the test helper plugin)
* Make sure the routines are still runned.

## UI changes

## Documentation

## Quality assurance


Fixes #
